### PR TITLE
Complying with eslint-config-standard (almost, 4spaces)

### PR DESCRIPTION
**Pull request by sadasant**
This PR showcases the power of eslint fix, how eslint-config-standard is not that different from our code, and how codacy works perfectly with these settings.

For a historical example of codacy failing the PR for eslint rules, check this: https://www.codacy.com/app/sadasant/futil-js/pullRequest?prid=523717

You can also see a failing example by clicking the commit f18afba, as it's visible in this picture:

[](https://i.imgur.com/eNd46Uq.png)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,9 +35,12 @@
     "codacy-coverage": "^2.0.1",
     "coveralls": "^2.11.16",
     "eslint": "^3.7.1",
-    "eslint-config-smartprocure": "^0.0.4",
+    "eslint-config-standard": "^6.2.1",
     "eslint-plugin-lodash-fp": "^2.0.1",
     "eslint-plugin-mocha": "^4.8.0",
+    "eslint-plugin-promise": "^3.4.2",
+    "eslint-plugin-standard": "^2.0.1",
+    "istanbul": "^0.4.5",
```
Comment (daedalus28) [inline package.json]: did you mean to add istanbul back?

**Comment by sadasant** [comment]
Lets merge thisss 😱❤

File: src/index.js [hunk 2]
```diff
@@ -22,4 +22,4 @@ export const flowMap = (...fns) => _.map(_.flow(...fns))
 // Misc
 // ----
 export const testRegex = regex => regex.test.bind(regex)
-export const compareDeep = _.curry((path, item, other) => _.get(path, item) == other)
+export const compareDeep = _.curry((path, item, other) => _.get(path, item) === other)
```
Comment (daedalus28) [inline src/index.js]: just realized this actually does need to be double equal so it doesn't change the behavior of the method

Comment (sadasant) [inline src/index.js]: Right! Let me fix it

**Pull request merged**